Library/ViewModels/DeprecatedRewardPledgeViewModel.swift

```diff
@@ -263,7 +262,7 @@ public final class DeprecatedRewardPledgeViewModel: Type, Inputs, Outputs {
     let shippingRules = shippingRulesEvent.values()
 
     self.navigationTitle = projectAndReward
-      .map(navigationTitle(forProject:reward:))
+      .map(title(forProject:reward:))
```

Comment:
Had to change the name of this `private` function...otherwise `SwiftFormat` would inset `self.` (probably false positive since the function name was identical to the property being assigned to)